In Solaris 11.4, we will be normalizing man pages. See PSARC 2015/428.
This means that what was section 4, is now section 5 and what was in
section 5 is now in section 7. But currently only for Solaris 11.4.

This patch should eventually be sent upstream, but for that to happen
it will need to be reworked into a form that will be acceptable for
all versions of Solaris. Until then, we are just using this Solaris 11.4
specific patch.

--- tcpdump-4.9.3/configure.ac	Thu Oct 24 13:15:24 2019 +0200
+++ tcpdump-4.9.3/configure.ac	Thu Oct 24 13:16:30 2019 +0200
@@ -721,12 +721,6 @@
 
 solaris*)
 	V_GROUP=sys
-
-	#
-	# Use System V conventions for man pages.
-	#
-	MAN_FILE_FORMATS=4
-	MAN_MISC_INFO=5
 	;;
 esac
 
